Automattic/wp-calypso
# FSE: Add the "Showcase New Blocks" module

#### Changes proposed in this Pull Request

128-gh-Dotcom-roadmap

#### Testing instructions

1. Start the FSE dev build:
   ```shell
   cd apps/full-site-editing
   nvm use
   yarn && yarn dev
   ```
2. Go to the editor,
3. Open the block inserter drop-down,
4. Under the "Most used" category, there should be a "New" category containing [6 blocks](https://github.com/Automattic/wp-calypso/pull/41822#discussion_r420993908):
   <p align="center">
     <img width="340" alt="Screenshot 2020-05-05 19 40 21" src="https://user-images.githubusercontent.com/1451471/81294952-ea022480-906f-11ea-8b40-2457ad9cc087.png">
   </p>

Fixes https://github.com/Automattic/Dotcom-roadmap/issues/128
